test/unit/meta/has_abi.cpp

```diff
@@ -15,34 +15,38 @@
 {
-  TTS_EXPECT_NOT(( eve::has_native_abi_v<eve::wide<double, eve::fixed<32>>>              ));
-  TTS_EXPECT_NOT(( eve::has_native_abi_v<eve::logical<eve::wide<double, eve::fixed<32>>>>));
+  constexpr auto native = eve::wide<float>::static_size;
+
+  TTS_EXPECT_NOT(( eve::has_native_abi_v<eve::wide<float, eve::fixed<2*native>>>              ));
+  TTS_EXPECT_NOT(( eve::has_native_abi_v<eve::logical<eve::wide<float, eve::fixed<2*native>>>>));
 
 #if defined(EVE_NO_SIMD)
-  TTS_EXPECT_NOT( eve::has_native_abi_v<eve::wide<double>>                );
-  TTS_EXPECT_NOT( eve::has_native_abi_v<eve::logical<eve::wide<double>>>  );
+  TTS_EXPECT_NOT( eve::has_native_abi_v<eve::wide<float>>                );
+  TTS_EXPECT_NOT( eve::has_native_abi_v<eve::logical<eve::wide<float>>>  );
 #else
-  TTS_EXPECT( eve::has_native_abi_v<eve::wide<double>>                );
-  TTS_EXPECT( eve::has_native_abi_v<eve::logical<eve::wide<double>>>  );
+  TTS_EXPECT( eve::has_native_abi_v<eve::wide<float>>                );
+  TTS_EXPECT( eve::has_native_abi_v<eve::logical<eve::wide<float>>>  );
 #endif
 
-  TTS_EXPECT( eve::has_native_abi_v<double>                           );
-  TTS_EXPECT( eve::has_native_abi_v<eve::logical<double>>             );
+  TTS_EXPECT( eve::has_native_abi_v<float>                           );
+  TTS_EXPECT( eve::has_native_abi_v<eve::logical<float>>             );
 }
 
 TTS_CASE( "Check for detection of aggregated ABI")
 {
+  constexpr auto native = eve::wide<float>::static_size;
+
 #if defined(EVE_NO_SIMD)
-  TTS_EXPECT_NOT(( eve::has_aggregated_abi_v<eve::wide<double, eve::fixed<32>>>              ));
-  TTS_EXPECT_NOT(( eve::has_aggregated_abi_v<eve::logical<eve::wide<double, eve::fixed<32>>>>));
+  TTS_EXPECT_NOT(( eve::has_aggregated_abi_v<eve::wide<float, eve::fixed<2*native>>>              ));
+  TTS_EXPECT_NOT(( eve::has_aggregated_abi_v<eve::logical<eve::wide<float, eve::fixed<2*native>>>>));
 #else
-  TTS_EXPECT(( eve::has_aggregated_abi_v<eve::wide<double, eve::fixed<32>>>              ));
-  TTS_EXPECT(( eve::has_aggregated_abi_v<eve::logical<eve::wide<double, eve::fixed<32>>>>));
+  TTS_EXPECT(( eve::has_aggregated_abi_v<eve::wide<float, eve::fixed<2*native>>>              ));
+  TTS_EXPECT(( eve::has_aggregated_abi_v<eve::logical<eve::wide<float, eve::fixed<2*native>>>>));
 #endif
 
-  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<eve::wide<double>>                );
-  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<eve::logical<eve::wide<double>>>  );
+  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<eve::wide<float>>                );
+  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<eve::logical<eve::wide<float>>>  );
 
-  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<double>                           );
-  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<eve::logical<double>>             );
+  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<float>                           );
+  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<eve::logical<float>>             );
```

Comment:
No idea what this is

Reply:
We check scalars don't get assigned the aggregated ABI
Those are old tests, but they check out ;)